doc/loanrules.md

```diff
@@ -1,19 +1,22 @@
 # Loan Rules
 
-The loan rules engine calculates the loan policy (that specifies the loan period)
-based on the patron's patron group and the item's material type, loan type, and location.
+The loan rules engine calculates the loan policies based on the patron's
+patron group and the item's material type, loan type, and location.
```

Comment:
Now we have a loan policy, a request policy and a notice policy.
Therefore my suggestion is to rename the engine from loan rules engine to circulation rules engine and the loan rules file to circulation rules file.

Reply:
I agree that it is worthwhile to review this. I believe that needs discussion and careful scheduling, and should not hold up this work.

I've raised https://issues.folio.org/browse/FOLIO-1789 to reflect that suggestion.